### ISSUE TITLE
Even simpler textarea composer

### ISSUE THREAD

**Issue by ara4n** [vector-im/riot-web#4861]
for those who are hitting weird bugs like custom keybindings being ignored by Draft

**Comment by lukebarnard1** [comment]
I'm not sure what disabling the RTE would look like...

**Comment by ara4n** [comment]
I've had another request (due to it interacting badly with a 'privacy' extension). Disabling RTE would mean keeping the old Composer around for a bit longer, but just not turned on by default.

**Comment by turt2live** [comment]
This seems like the best releated issue, but [someone in #riot](https://matrix.to/#/!DgvjtOljKujDBrxyHk:matrix.org/$150440524577oWmQg:matrix.rkm.id.au) would like to keep the emoji autocomplete, but not have the formatting keybindings. Would this solve that?

**Comment by eqyiel** [comment]
Hey, I'm that someone from #riot.

The RTE is weird for people using MacOS/GTK with Emacs/readline keys.

For example, C-a should go to the start of the line, instead it highlights everything in the text field.

There are a bunch more like this.

**Comment by lukebarnard1** [comment]
>keeping the old Composer around for a bit longer, but just not turned on by default.

I'd be more inclined to try and make the new one work by improving it, perhaps optimising for customisation of keyboard shortcuts if that's what people are wanting the most.

**Comment by ara4n** [comment]
> I'd be more inclined to try and make the new one work by improving it,

I think this is going to be tough. Many of the complaints here are folks who had browser extensions which swap `<textarea/>` for some other richer editor.  This one (custom keyboard shortcuts) is a bit more obscure.  Can we just retrofit the textarea implementation into the new composer for folks who don't want fullblown Draft?

**Comment by lukebarnard1** [comment]
Sure, I think we can still provide some of the auto-completion functionality whilst using a `<textarea/>`. Pills would be problematic.

**Comment by ara4n** [comment]
Yeah, i'm assuming pills and RTE(!) would disappear, but ideally keep the autocomplete UI.

**Comment by eqyiel** [comment]
Another one of these: https://github.com/vector-im/riot-web/issues/5165

**Comment by non-Jedi** [comment]
Fixing this probably also fixes #4936 and #4909. Possibly others I'm missing.

**Comment by t3chguy** [comment]
Integrating autocomplete with textarea is somewhat more annoying than I had envisioned
but it is nearly done!!

**Comment by turt2live** [comment]
@ara4n is this still a requirement given we now have slate and RTE/markdown are different modes?

**Comment by t3chguy** [comment]
There is no longer an RTE.

**Comment by non-Jedi** [comment]
@t3chguy the reason for this issue (behind the actual name) is people who use browser extensions that rely on there being a `textarea` for editable text. That is still a problem with the current Riot UI.
